retrieve_oa_files.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python 3.5
+
+import sys, os, glob, shutil, logging, time, argparse, glob
+from datetime import datetime
+
+def getAppIDs(fname,series):
+    try:
+        with open(os.path.abspath(fname)) as fd:
+            for line in fd:
+                if line.startswith(series):
+                    appids.append(line.strip())
+        logging.info("-- Number of appids for this series: "+str(len(appids)))
+    except IOError as e:
+         logging.error('-- I/O error({0}): {1}'.format(e.errno,e.strerror))
+    except:
+         logging.error('-- Unexpected error:', sys.exc_info()[0])
+         raise
+
+def constructPath(appid):
+    series = appid[:2]
+    series2 = appid[2:5]
+    series3 = appid[5:8]
+    fileurl = oafilespath+'\\'+series+'\\'+series2+'\\'+series3
```

Comment:
I would probably do this as a one-liner-- and since it's only used once, you could even do it inline where it's needed:

    return os.path.join(oafilespath, appid[:2], appid[2:5], appid[5:8])